templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.tikaConfig }}
```

Comment:
@ahilmathew please add Apache License v2 file header. Thank you

Reply:
@lewismc Done.